src/WebApp/WebApp.csproj

```diff
@@ -30,6 +30,9 @@
     <EmbeddedResource Update="Resources\Pages\QuestionsModel.en.resx">
       <Generator>PublicResXFileCodeGenerator</Generator>
     </EmbeddedResource>
+    <EmbeddedResource Update="Resources\Pages\Shared\_Header.en.resx">
+      <Generator>PublicResXFileCodeGenerator</Generator>
+    </EmbeddedResource>
     <EmbeddedResource Update="Resources\Pages\Shared\_Layout.en.resx">
       <Generator>PublicResXFileCodeGenerator</Generator>
     </EmbeddedResource>
```

Comment:
You deleted this file, it should no longer be referenced.

Reply:
Its still there. Has like one line of translation on it

Reply:
Oh, I saw that incorrectly, you can mark this as resolved.